docs/about/distro.md

```diff
@@ -22,7 +22,7 @@ For major feature differences, the following chart may be helpful.
 |------------------------------------------------|------------------------------------------------------------------|----------------------|------------------|-------------------------------------|-------------------------------------|-------------------------------------|-------------------------------------|-------------------------------------|-------------------------------------|
 | [OmniOS CE](https://www.omniosce.org/)         | [OmniOS Community Association](https://omniosce.org/about/about) | Servers              | Debian           | <i class="material-icons">check</i> | IPS                                 | <i class="material-icons">check</i> |                                     | <i class="material-icons">check</i> | <i class="material-icons">check</i> |
 | [OpenIndiana](https://www.openindiana.org/)    | [OpenIndiana](https://www.openindiana.org/)                      | Workstation, Servers | Ubuntu           | <i class="material-icons">check</i> | IPS                                 | <i class="material-icons">check</i> |                                     | <i class="material-icons">check</i> |                                     |
-| [SmartOS](https://smartos.org)                 | [Joyent](https://www.joyent.com)                                 | Hypervisor           | CoreOS           | <i class="material-icons">check</i> | [pkgsrc](https://pkgsrc.joyent.com) | <i class="material-icons">check</i> |                                     | <i class="material-icons">check</i> | <i class="material-icons">check</i> |
+| [SmartOS](https://smartos.org)                 | [MNX Solutions](https://mnx.io/)                                 | Hypervisor           | CoreOS           | <i class="material-icons">check</i> | [pkgsrc](https://pkgsrc.joyent.com) | <i class="material-icons">check</i> |                                     | <i class="material-icons">check</i> | <i class="material-icons">check</i> |
```

Comment:
Make the URL point to https://mnxsolutions.com instead  please.  :)

Reply:
Done